src/Pages/settings/Preferences.tsx

```diff
@@ -0,0 +1,55 @@
+import { useDispatch, useSelector } from "react-redux";
+import { setPreferences, UserPreferences } from "State/Login";
+import { RootState } from "State/Store";
+import "./Preferences.css";
+
+const PreferencesPage = () => {
+    const dispatch = useDispatch();
+    const perf = useSelector<RootState, UserPreferences>(s => s.login.preferences);
+
+    return (
+        <div className="preferences">
+            <h3>Preferences</h3>
+
+            <div className="card flex">
+                <div className="flex f-col f-grow">
+                    <div>Theme</div>
+                </div>
+                <div>
+                    <select value={perf.theme} onChange={e => dispatch(setPreferences({ ...perf, theme: "light" === e.target.value ? "light" : "dark" }))}>
```

Comment:
Wdyt about keeping current default (use `system` theme)? If user explicitly sets a theme we can override it but if not use system theme. Have my devices configured to use dark/light depending on time of the day, don't want to be manually changing it from preferences.

This would be similar to other preferences, keeping the current default but offering customization.